### ISSUE TITLE
Some of functions with inconsistent argument names

### ISSUE THREAD

**Issue by kaiidams** [dotnet/TorchSharp#744]
Some functions have `inPlace` and others `inplace` while they have `inplace` in PyTorch.

```
                static public Tensor dropout(Tensor input, double probability = 0.5, bool training = true, bool inPlace = false)
                static public Tensor Hardtanh(Tensor x, double min_val = -1.0, double max_val = 1.0, bool inplace = false)
```

Some functions have `probability` while they have `p` in PyTorch.

```
                static public Tensor dropout(Tensor input, double probability = 0.5, bool training = true, bool inPlace = false)
```

Some functions are capitalized, others are lowercased while they are lowercased in PyTorch.

```
                static public Tensor GLU(Tensor x, int dim = -1)
                static public Tensor gelu(Tensor x)
```

**Comment by kaiidams** [comment]
Some have `dimension`, while others have `dim` while they have `dim` in PyTorch.

```
                static public Tensor log_softmax(Tensor x, long dimension)
                static public Tensor softmax(Tensor x, long dim)

**Comment by NiklasGustafsson** [comment]
I'll try to go through them. Unfortunately, PT itself is not consistent, especially on p/probability and dim/dimensions
